certified-connectors/shared_hvi-20ins-20specific-20vehicle-5fbc7b7a898d7451-02a9ed7ac5553fca/settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@jrsinnovation Please remove the icon.png and settings.json files as they are not required in GitHub.